posthog/hogql/property.py

```diff
@@ -70,7 +70,7 @@ def visit_call(self, node: ast.Call):
 def property_to_expr(
     property: Union[BaseModel, PropertyGroup, Property, dict, list, ast.Expr],
     team: Team,
-    scope: Literal["event", "person", "session", "replay", "replay_entity"] = "event",
+    scope: Literal["event", "person", "session", "replay", "replay_entity", "replay_pdi"] = "event",
```

Comment:
I'm a little worried about all the scopes we're adding here. I think we're somewhat fighting the system by adding all these different scopes but I don't fully understand this function enough yet to say that definitively.

I've cleaned up some of this (by adding yet another scope in https://github.com/PostHog/posthog/pull/22750) but will look at it holistically once we're removed all of our old filtering code and hopefully only have a single scope (`recordings`) here.